book.js

```diff
@@ -0,0 +1,60 @@
+// ─── Selectors ───────────────────────────────────────────────────────────────
+const bookTitle = document.querySelector('#bookTitle');
+const bookAuthor = document.querySelector('#bookAuthor');
+const addBook = document.querySelector('#addBook');
+const lists = document.querySelector('.lists');
+
+// ─── Functions ───────────────────────────────────────────────────────────────
+
+const bookCollection = [];
+
+function removeBooks() {
+  const el = document.querySelector(`.${this.id}`);
+  el.remove();
+}
+
+function createBook() {
+  let listOfBooks = [];
+
+  if (localStorage.getItem('bookCollection') !== null) {
+    listOfBooks = JSON.parse(localStorage.getItem('bookCollection'));
+  }
+  const wrapper = document.createElement('div');
+  const p1 = document.createElement('p');
+  const p2 = document.createElement('p');
+  const line = document.createElement('hr');
+  const removeBtn = document.createElement('button');
+  wrapper.className = 'container';
+  p1.setAttribute('id', 'booktitle');
+  p2.setAttribute('id', 'bookauthor');
+  removeBtn.textContent = 'Remove';
+  for (let i = 0; i < listOfBooks.length; i += 1) {
+    p1.textContent = listOfBooks[i].title;
+    p2.textContent = listOfBooks[i].author;
+    wrapper.className = `btn-${i}`;
+    removeBtn.setAttribute('id', `btn-${i}`);
+  }
+  wrapper.append(p1, p2, removeBtn, line);
+  removeBtn.addEventListener('click', removeBooks);
+
+  lists.prepend(wrapper);
+}
+
+function addingBook() {
+  const book = { title: '', author: '' };
+  book.title = bookTitle.value;
+  book.author = bookAuthor.value;
+  bookCollection.push(book);
+  if (localStorage.getItem('bookCollection') === null) {
+    localStorage.setItem('bookCollection', '[]');
+  }
```

Comment:
- [ ] Great job working on your project :+1: I noticed that when I tried to add a book for the first time, the book didn't get added but on the second try, the book was added successfully. This might be a result of a bug in your project so kindly fix this to enhance user interactivity :100: I have attached a video below for your reference. :+1: 
- Reference Video
[Screencast from 02-14-2023 06:26:54 PM.webm](https://user-images.githubusercontent.com/102699958/218825352-f7e792f5-34cf-418b-9526-38c900608720.webm)